src/app/components/dialog-lost/dialog-lost.component.html

```diff
@@ -0,0 +1,18 @@
+<h1 mat-dialog-title class="text-center">Perdiste!!!</h1>
```

Comment:
no creeis un nuevo componente dialog, lo que tenéis que hacer es borrar dialog-lost y dialog-win y crear un componente genérico. Los componentes no van asociados a una funcionalidad concreta como ganar o perder, eso son reglas, los componentes deberían aceptar parámetros como el texto de descripción, el texto del botón y la función que debe de ejecutar ese botón: string, string, callback, por ejemplo